packages/zowe-explorer/__tests__/__unit__/extension.unit.test.ts

```diff
@@ -464,7 +470,7 @@ describe("Extension Unit Tests", () => {
 
         expect(globals.ISTHEIA).toEqual(true);
         // tslint:disable-next-line: no-magic-numbers
-        expect(globalMocks.mockMkdirSync.mock.calls.length).toBe(4);
+        expect(globalMocks.mockMkdirSync.mock.calls.length).toBe(6);
```

Comment:
we should probably remove tslint in favor of eslint.
and for these kind of tests, we could have the eslint configuration for tests exclude the magic-number rules 😋 
Example: https://github.com/zowe/zowe-cli/blob/master/.eslintrc.js#L14-L36

Reply:
it doesn't look like Zowe Explorer has changed over to eslint, only ZE API & FTP

Reply:
🤦 you are right
the first step is to migrate to eslint 😋